misc/pom.xml

```diff
@@ -17,7 +17,7 @@
   <parent>
     <groupId>org.apache.ratis</groupId>
     <artifactId>ratis-thirdparty</artifactId>
-    <version>1.0.1</version>
+    <version>1.1.0-SNAPSHOT</version>
```

Comment:
Bump version to verify locally. I'm unsure whether it's proper to be included in this patch.

Reply:
@tisonkun , thanks a lot for working on this!  Please don't change the version. We will let the release managers to change it.  The change looks good other than that.

Reply:
@szetszwo OK. Updated. Please give it another pass :)